R/create_value_box_grid.R

```diff
@@ -0,0 +1,28 @@
+#' Internal helper for creating grid of `valuboxes`
+#' for displaying information.
+#'
+#' @keywords internal
+create_value_box_grid = function(df) {
+  vbs = lapply(names(df), function(col_name) {
+    bslib::value_box(
+      title = htmltools::tags$span(col_name, style = "font-size:12px;"),
+      value = htmltools::tags$span(as.character(df[[1, col_name]]), style = "font-size:20px;"),
```

Comment:
issue: Use relative units, they are easier to manage.

```suggestion
      title = htmltools::tags$span(col_name, style = "font-size:10pt;"),
      value = htmltools::tags$span(as.character(df[[1, col_name]]), style = "font-size:16pt;"),
```